### DISCUSSION TITLE
Is there a way to get last_four_digits of the card entered within CardPayment brick

### DISCUSSION THREAD

**Discussion by joxpulp** [mercadopago/sdk-react#64] [Q&A] [answered]
I wonder if there is an way to get the last 4 digits of the card entered within CardPayment brick, this is interesting cause i want to show a summary before the payment and show the card type and last four digits. thanks

**Comment by francineemilia** [comment, marked as answer]
Hi @joxpulp, We are currently unable to provide the last four, however we have added your suggestion to our improvement backlog! Thanks for you idea and feedback.
